compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirMemberPropertyChecker.kt

```diff
@@ -111,11 +111,11 @@ object FirMemberPropertyChecker : FirRegularClassChecker() {
     private fun checkAccessor(
         accessor: FirPropertyAccessor?,
         delegate: FirExpression?,
-        report: (FirSourceElement, FirPropertyAccessorSymbol) -> Unit,
+        report: (FirSourceElement, FirPropertyAccessorSymbol, hasBody: Boolean) -> Unit,
```

Comment:
Nit: You don't need the parameter name (`hasBody:`) in the signature

Reply:
This is just to make the code more readable.